pymongo/asynchronous/encryption.py

```diff
@@ -398,6 +398,7 @@ def __init__(self, client: AsyncMongoClient[_DocumentTypeArg], opts: AutoEncrypt
             encrypted_fields_map = _dict_to_bson(opts._encrypted_fields_map, False, _DATA_KEY_OPTS)
         self._bypass_auto_encryption = opts._bypass_auto_encryption
         self._internal_client = None
+        opts._kms_ssl_contexts(_IS_SYNC)
```

Comment:
It's worth adding a comment here to explain that we call this here so that parsing errors can be raised before creating internal clients.

Reply:
makes sense, added :)